third-party/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 string(REPLACE ${PROJECT_SOURCE_DIR}/ "" _rel_path ${CMAKE_CURRENT_LIST_DIR})
 
-include(CMake/external_json.cmake)
 
 add_subdirectory( "${CMAKE_CURRENT_LIST_DIR}/rsutils" )
 
+pop_security_flags()
```

Comment:
Please add a comment about why we remove security flags here,
Indicating that we cannot guarantee third party security enforcements